tests/string-util.cpp

```diff
@@ -0,0 +1,63 @@
+#include <iostream>
+#include <assert.h>
+#include <string.h>
+
+#include "gamestruct/string-utility.hpp"
+#include "datastruct/vector.hpp"
+
+using namespace datastruct;
+struct functionMETA {
+    void (*funcPtr) ();
+    const char *funcName;
+};
+
+namespace test {
+    void copyString() {
+        char* str = "Hello World";
+        char* str1=stringUtility::copyString(str);
+        assert(strcmp(str,str1)==0); // TODO: Non riesce a darmi un ❌, mi sa che deve essere sovrascritto, un wrapper ad assert?
+        delete[] str1;
+        str1=stringUtility::copyString(str,5);
+        str = "Hello";
+        assert(strcmp(str,str1)==0); // TODO: Non riesce a darmi un ❌, mi sa che deve essere sovrascritto, un wrapper ad assert?
+        delete[] str1;
+    }
+    void repeatChar() {
+        char* str = "aaaaaaaaaa";
+        char* str1=stringUtility::repeatChar(10,'a');
+        assert(strcmp(str,str1)==0); // TODO: Non riesce a darmi un ❌, mi sa che deve essere sovrascritto, un wrapper ad assert?
+        delete[] str1;
+    }
+    void splitString() {
+        char* str = "abcdefghijklmnopqrstuvwxyz";
+        char* array[6]= {"abcde","fghij","klmno","pqrst","uvwxy","z"};
```

Comment:
🟥 
il compilatore si lamenta da me 
`./tests/string-util.cpp:33:69: error: ISO C++ forbids converting a string constant to ‘char*’ [-Werror=write-strings]`

Reply:
❓  ma comunque non sarebbe stato più chiaro mettere la roba nel test in un altra PR?